src/NzbDrone.Core/Extras/ExistingExtraFileService.cs

```diff
@@ -10,7 +10,7 @@ namespace NzbDrone.Core.Extras
 {
     public interface IExistingExtraFiles
     {
-        List<string> ImportExtraFiles(Series series, List<string> possibleExtraFiles);
+        List<string> ImportExtraFiles(Series series, List<string> possibleExtraFiles, bool import = true);
```

Comment:
Instead of the very generic `import` indicating what is calling it, this chould be renamed to how it will be used, something like `cleanRemovedFiles` or a better name for whether all series files are being passed in, maybe something like `possibleExtraFilesForEntireSeries`.

 I also don't think it needs a default value since it's only used in a couple places.